build/xtask/src/dist.rs

```diff
@@ -924,7 +924,7 @@ fn generate_task_linker_script(
     fn emit(linkscr: &mut File, sec: &str, o: u32, l: u32) -> Result<()> {
         writeln!(
             linkscr,
-            "{} (rwx) : ORIGIN = 0x{:08x}, LENGTH = 0x{:08x}",
+            "{} (rwx) : ORIGIN = {:#08x}, LENGTH = {:#08x}",
```

Comment:
```suggestion
            "{} (rwx) : ORIGIN = {:#010x}, LENGTH = {:#010x}",
```

Reply:
I'll stop repeating this, but all the ones with explicit field widths need 2 added to them; I think the field width is including the `0x` in its calculation after the change to use `#`.